cmd/release/main.go

```diff
@@ -0,0 +1,51 @@
+// Copyright (C) 2019 Storj Labs, Inc.
```

Comment:
i think this command (go-release) could be a bash script? 

```
#!/bin/bash
TIMESTAMP=$(date -Is)
COMMIT=$(git rev-parse HEAD)
VERSION=$(git describe --tags --match “v[0-9]*.[0-9]*.[0-9]*”)
gocommand=$1
shift
exec go "$gocommand" -ldflags "-X storj.io/storj/internal/version.Timestamp=$TIMESTAMP -X storj.io/storj/internal/version.CommitHash=$COMMIT -X storj.io/storj/internal/version.Version=$VERSION storj.io/storj/internal/version.Release=true" "$@"
```

Reply:
I can put those commands into this file and have unified 👍

Reply:
maybe instead of that with the `shift` it would be better to do

```
#!/bin/bash
TIMESTAMP=$(date -Is)
COMMIT=$(git rev-parse HEAD)
VERSION=$(git describe --tags --match “v[0-9]*.[0-9]*.[0-9]*”)
gocommand="$1"
exec go "$gocommand" -ldflags "-X storj.io/storj/internal/version.Timestamp=$TIMESTAMP -X storj.io/storj/internal/version.CommitHash=$COMMIT -X storj.io/storj/internal/version.Version=$VERSION storj.io/storj/internal/version.Release=true" "${@:2}"
```
and then make sure to run it through shellcheck lol

Reply:
Are you saying instead of a release go binary we do it via shell? I'm 100% behind that idea if so

Reply:
oh, yes!